app/about/about-page-context.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
```

Comment:
Hmmm so my thoughts on this are that it may be overkill. The about page in general is mainly informational and I am not sure we need to have a context for it (I assume this is to save their last visted/currentPage?) - I see the value in that conceptually but I wonder if there is even a use case for it.

If I am missing something else here, let me know.